src/bidiMapper/modules/cdp/CdpTarget.ts

```diff
@@ -134,8 +145,17 @@ export class CdpTarget {
   async #unblock() {
     try {
       await Promise.all([
+        (async () => {
+          await this.#cdpClient.sendCommand('Page.enable');
+          // There can be some existing frames in the target, if reconnecting to an
+          // existing browser instance, e.g. via Puppeteer. Need to restore the browsing
+          // contexts for the frames to correctly handle further events, like
+          // `Runtime.executionContextCreated`.
+          const frameTree =
```

Comment:
this is also incorrect: getFrameTree will only be sent once Page.enable is resolved, which is too late

Reply:
ack